CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# context-template ChangeLog
+# ed25519-signature-2020-context ChangeLog
```

Comment:
```suggestion
# cryptosuite-context-ed25519-2020 ChangeLog
```

See comments here for reasoning: https://github.com/digitalbazaar/ed25519-signature-2018-context/pull/1#discussion_r589657249